dts/bindings/sensor/st,lis2dh-common.yaml

```diff
@@ -20,8 +19,24 @@ properties:
 
     anym-on-int1:
       type: boolean
-      required: false
       description: |
         Indicates that the device driver should use interrupt 1
         for any movement. This is for boards that only have one
         interrupt line connected from the sensor to the processor.
+
+    anym-no-latch:
+      type: boolean
+      description: |
+        Disable the latching of interrupts for any movement.
+
+    anym-mode:
+      type: int
+      required: false
+      default: 0
```

Comment:
The default needs justification in the description. Is it the power on reset value?

Reply:
Yes, and this is added so we don't break existing boards.